.github/workflows/build_package.yml

```
@@ -0,0 +1,17 @@
+name: Build Maven Package
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
```

```suggestion
      - name: Set up JDK 1.11
```
Is this for backward compability reasons? I thought we agreed on JDK 11